src/TrueLayer/Payments/Model/PaymentUser.cs

```diff
@@ -3,5 +3,5 @@ namespace TrueLayer.Payments.Model
     /// <summary>
     /// Represents an end user
     /// </summary>
-    public record PaymentUser(string Id, string? Name = null, string? Email = null, string? Phone = null);
+    public record PaymentUser(string Id);
```

Comment:
This is a breaking change for the consumers of the SDK as the data is currently returned in the `GetPaymentResponse` object. Can you please link the API documentation in the PR and link any public change log stating that the fields get removed? Or, are we sure that those fields never received any value?

Reply:
The change log and API doc update are on staging. Will share the links here as soon as they are available in production. 

Currently, those fields may contain a value but we would like to stop returning them. The data should be available to the client internally as it's submitted to us by the client as part of the payment creation. 

 This is done to reduce sensitive information exchanges